src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -58,9 +141,30 @@ private static UIElement CreateElement(string text, double lineHeight, TextForma
                 Padding = new Thickness(1),
                 VerticalAlignment = VerticalAlignment.Center,
             };
-
             border.Measure(new Size(double.PositiveInfinity, double.PositiveInfinity));
             return border;
         }
+
+        /// <summary>
+        /// Determines if the border is being moused over and shows the info accordingly
+        /// </summary>
+        private void Border_ToolTipOpening(object sender, ToolTipEventArgs e)
+        {
+            var border = sender as Border;
```

Comment:
```suggestion
            var border = (Border)sender;
```

If the sender isn't the border for some reason, we have bigger problems. This cast will just mean we null ref which isn't what we want.